pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "phenopy"
+version = "0.5.3"
+description = "Phenotype comparison scoring by semantic similarity."
+authors = [
+    "Kevin Arvai <karvai@genedx.com>",
+    "Kyle Retterer <retterer@genedx.com>",
+    "Carlos Borroto <cborroto@genedx.com>",
+    "Vlad Gainullin <vgainullin@genedx.com>",
+    "Vincent Ustach <vustach@genedx.com>"
+]
+readme = "README.md"
+license = ""
+
+[tool.poetry.scripts]
+phenopy = "phenopy.__main__:main"
+
+[tool.poetry.urls]
+homepage = "https://github.com/GeneDx/phenopy"
+"Bug Tracker" = "https://github.com/GeneDx/phenopy/issues"
+
+[tool.poetry.dependencies]
+python = "^3.7"
+fire = "*"
+gensim = "*"
+networkx = "*"
+numpy = "*"
+obonet = "*"
+pandas = "^1.0.0"
+joblib = "*"
+scipy = "*"
+lightgbm = "3.0.0"
+requests = "*"
+ruff = "*"
```

Comment:
ruff, pytest, pytest-cov, and pre-commit more properly belong in the so called dev dependencies.

I'd remove them from here and add them with 
```
poetry add --group dev pytest pytest-cov ruff pre-commit
```
(though precommit is another thing I like to install globally with pipx :P )

Then any consumers of this library won't need to add these as dependencies.